src/services/merchantWebhookQueue.service.ts

```
@@ -78,63 +78,144 @@ export class MerchantWebhookQueueService {
+  /**
+   * Helper method to update webhook event status and details
+   */
+  private async updateWebhookEventStatus(
+    jobId: string,
+    status: MerchantWebhookEventEntityStatus,
+    attemptsMade: number,
+    nextRetry: Date | null | undefined,
+    error: string | null,
+    completedAt?: Date
+  ) {
+    const updateData: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)